CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### 1.2.6 (2019-12-23)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/app/PeterNgTr/codeceptjs-bshelper/pullRequest?prid=4722635)